go.mod

```diff
@@ -56,6 +56,7 @@ require (
 	github.com/hashicorp/terraform-svchost v0.1.1 // indirect
 	github.com/hashicorp/yamux v0.1.1 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
+	github.com/kisielk/errcheck v1.6.3 // indirect
```

Comment:
Why this requirement was added?

Reply:
Hmm, looks like it's automatically added by ide.
I don't touch requirements

Reply:
but you committed it, why then?